### PR TITLE
update requirements.txt with flake8, add .flake8 config and add linti…

### PR DESCRIPTION
**Pull request by mathagician**

### CODE REVIEW

File: oshc/main/templates/contest_submission.html [hunk 1]
```diff
@@ -3,8 +3,8 @@
 {% block content %}
 
 <div class="jumbotron text-xs-center">
-  <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
+  <div class="container text-center contest-submission-message">
+      <h1 class="display-3">Thank you!</h1>
```
Comment (tapaswenipathak) [inline oshc/main/templates/contest_submission.html:7]: 👋 You are trying to resolve two issues in this pull request. :)

File: .travis.yml [hunk 2]
```diff
@@ -23,6 +23,7 @@ before_script:
 
 script:
   - coverage run --source='main' manage.py test --verbosity 2
+  - flake8 --ignore E226,E302,E41,F401,E402 .
```
Comment (tapaswenipathak) [inline .travis.yml:26]: We'll also check these. Can you please remove the ignored ones.

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,13 @@
+configparser==3.5.0
 coverage==4.4.1
 dj-database-url==0.4.2
 Django==1.11.6
+enum34==1.1.6
+flake8==3.4.1
```
Comment (tapaswenipathak) [inline requirements.txt:6]: We can just have flake8.

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]
Thanks for the work! I have added few comments after that we are good to go!

**Comment by mathagician** [comment]
ok I shall give an updated request one for each issue.

**Comment by tapaswenipathak** [comment]
One way is to push changes to this pull request, then squash them later. Other very easy and not so good way is to close this pull request and create a new one with changes and suggestions accommodated. Let us know if you face issues while trying out these.

**Comment by mathagician** [comment]
I have given a new PR at https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/pull/178 

Please see and review.

**Review by tapaswenipathak — COMMENTED** [review]
Closing this one as you have submitted  #178.

**Pull request closed without merging**